Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Beckn/OnUpdate.hs

```diff
@@ -364,6 +364,7 @@ onUpdate ValidatedBookingCancelledReq {..} = do
   fork "incrementing fraud counters" $ do
     mFraudDetected <- SMC.anyFraudDetected booking.riderId booking.merchantId merchantConfigs
     whenJust mFraudDetected $ \mc -> SMC.blockCustomer booking.riderId (Just mc.id)
+  -- TODO  If Ride doesn't exist in BAP, create a cancelled ride entry.
```

Comment:
@roman-bodavskiy Did you miss this todo by mistake ?
OR you will do the changes later ?

Reply:
@RitikaHanish this TODO require changes in on_update api. Working on this now. Can be merged in separate PR